### PR TITLE
Sphinx docs

### PR DESCRIPTION
**Pull request by benauthor**
@lapointexavier trying to improve docs, what do you think http://percolate.github.io/bfh/

### CODE REVIEW

File: bfh/__init__.py [hunk 1]
```diff
@@ -252,17 +252,17 @@ class DogToAnimal(Mapping):
             legs = Const(4)
             noise = Const('woof!')
 
-    The action happens when you get an instance of your mapping:
+    The action happens when you get an instance of your mapping::
 
         dog_to_animal_map = DogToAnimal()
 
-    Then apply it to an object. You get a Schema instance:
+    Then apply it to an object. You get a Schema instance::
```
Comment (lapointexavier) [inline bfh/__init__.py:259]: Just curious, why `::` ?

**Comment by lapointexavier** [comment]
@benauthor Looks great! Is there a way to include the README content from the github page too, if it make sense to have it there. It provides some general ideas / use cases.

Also wondering, is the _static folder missing? Maybe it wasn't pushed yet, if so ignore my comment.

File: bfh/__init__.py (hunk 1 above)
Comment (benauthor) [inline bfh/__init__.py:259]: That's an rst literal block http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks

**Comment by benauthor** [comment]
Static is autogenerated by sphinx and ends up in doc/site/_static, which it looks like jekyll auto-ignores because it's underscore-prefixed... 😛

**Comment by lapointexavier** [comment]
Very nice. 🐑

**Pull request merged**